packages/perseus/src/util/parse-perseus-json/regression-tests/parse-perseus-json-regression.test.ts

```diff
@@ -0,0 +1,20 @@
+import * as fs from "fs";
+import {join} from "path";
+
+import {parseAndTypecheckPerseusItem} from "../index";
+
+const dataFiles = fs.readdirSync(join(__dirname, "data"));
```

Comment:
Nice. So adding a regression test in the future is as easy as creating a file in `regression-tests/data` and checking it in, right?